packages/charts/react-charting/src/components/DeclarativeChart/PlotlySchemaAdapter.ts

```diff
@@ -94,6 +94,9 @@ export const transformPlotlyJsonToDonutProps = (
     hideLabels,
     showLabelsInPercent: firstData.textinfo ? firstData.textinfo === 'percent' : true,
     styles,
+    legendProps: {
+      canSelectMultipleLegends: true,
+    },
```

Comment:
legend props are defined in DeclarativeChart.tsx. Use that only.

Reply:
done